src/nvim/map.h

```diff
@@ -42,6 +42,7 @@ MAP_DECLS(handle_T, ptr_t)
 MAP_DECLS(String, MsgpackRpcRequestHandler)
 MAP_DECLS(HlEntry, int)
 MAP_DECLS(String, handle_T)
+MAP_DECLS(uint64_t, cstr_t)
```

Comment:
redundant, on RHS a string is just a pointer (probably not used here anyway anymore)